src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js

```diff
@@ -63,7 +61,7 @@ function KeyBasedAssetTransferIntro() {
         backgroundColor={darkTheme.colors.basic090}
       >
         <Content>
-          <Spacing h={top} />
+          <Spacing h={60} />
```

Comment:
If I understand the issue correctly the problem is that top of the screen has a different shade of black there. 
Simple solution would be to change `darkTheme.colors.basic090` to `darkTheme.colors.background`.

Proper solution would be to create a new floating back button component that will allow content to flow underneath itself.

Proposed solution of fixed top spacing does not seem to address that.

Reply:
@mdjastrzebski @IAmKio @poocart Done.